upload/src/addons/SV/ThreadReplyBanner/XF/Pub/Controller/Forum.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SV\ThreadReplyBanner\XF\Pub\Controller;
+
+use XF\Mvc\ParameterBag;
+
+class Forum extends XFCP_Forum {
+
+	protected function setupThreadCreate(\XF\Entity\Forum $forum)
+	{
+		/** @var \SV\ThreadReplyBanner\XF\Service\Thread\Creator $creator */
+		$creator = parent::setupThreadCreate($forum);
+
+		$replyBanner = $this->filter('thread_reply_banner', 'str');

```

Comment:
This needs to check `canManageThreadReplyBanner()` to verify the user is allowed to create a banner